Remarkable-Donkeys/bangazon
# Kn setup

# Steps To Test #
1. Check to make sure Payment Type Model looks correct
2. Check to make sure Training Program Type looks correct


[Payment Type](https://github.com/Remarkable-Donkeys/bangazon/issues/3)
[Training Program](https://github.com/Remarkable-Donkeys/bangazon/issues/9)

### Description of what should happen when steps are followed ###
This is an initial pull request to provide models to teammates. Please take a look to see if anything looks wrong.
**At least one person needs to approve pull request before merged to master**
